bot/bot_secrets.py

```diff
@@ -21,6 +22,10 @@ def __init__(self) -> None:
         self._error_log_channel_ids: list[int] | None = None
         self._class_archive_category_ids: list[int] | None = None
         self._class_notifs_channel_id: int | None = None
+        self._birthday_role_id: Optional[int] = None
+        self._birthday_cooldown_in_days: Optional[int] = None
+        self._birthday_channel_id: Optional[int] = None
+        self._birthday_announcement_mode: Optional[int] = None
```

Comment:
Instead of optional we use `| None` which conveys the same thing. It's nicer and a little shorter.